composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle" : "~2.3",
-        "fsi/admin-bundle" : "1.1.x-dev"
+        "symfony/framework-bundle" : "^2.3",
+        "fsi/admin-bundle" : "~1.1@dev"
```

Comment:
when I look at this I start thinking about creating tagi 1.1.0 on admin-bundle branch 1.1.